go.mod

```diff
@@ -21,14 +20,81 @@ require (
 	k8s.io/api v0.18.2
 	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6 // indirect
-	k8s.io/kube-proxy v0.18.3 // indirect
 	k8s.io/kubectl v0.17.4
 	sigs.k8s.io/controller-runtime v0.5.2
 )
 
+require (
```

Comment:
Do we run "go mod tidy" to remove unnecessary modules, it looks like there are many module changes.